scripts/can_errgo/Demos/listenerDemo.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"encoding/binary"
+	"fmt"
+	"log"
+	"net"
+	"golang.org/x/sys/unix"
+)
+
+const (
+	canInterface = "vcan0"  // Use the virtual CAN interface
+	canFrameSize = 16       // CAN frame size (for a standard CAN frame)
+)
+
+func IntPow(n, m int) int {
```

Comment:
isn't there a built in function for this?

Reply:
looks like you don't use this function anywhere. it can be removed

Reply:
I believe there is an built in function for float64. Anyhow, I have removed the function.